### ISSUE TITLE
Roadmap

### ISSUE THREAD

**Issue by brannonh** [Dyldog/luhman-obsidian-plugin#19]
Looking for input, as far as a roadmap is concerned. My thought is to release a version `1.1.1` as a general bug fix release, which is well on its way. Passed that, I'm open to suggestions.

## Planned

- `1.1.1` - Bug Fixes

## Future

**Comment by Eforen** [comment]
What do you think of also including 

[FEATURE] Create zettel child/singling without opening new note #22 ?

**Comment by Eforen** [comment]
Also I am gona try to work on making sure everything has a config option though that may mean it should have an advanced section of the config not just throw everything at people so they don't feel overwhelmed at start.

**Comment by Eforen** [comment]
I edited your original post to include that we are fixing the bug with the links

**Comment by brannonh** [comment]
> What do you think of also including
> 
> [FEATURE] Create zettel child/singling without opening new note #22 ?

My preference would be to focus on bugs for 1.1.1 and start adding new features in the next release.

**Comment by brannonh** [comment]
@Dyldog Any chance we could get milestones enabled? 🙂

**Comment by Eforen** [comment]
> > What do you think of also including
> > [FEATURE] Create zettel child/singling without opening new note #22 ?
> 
> My preference would be to focus on bugs for 1.1.1 and start adding new features in the next release.

Fair point! In that case get on thos PRs I put in so we can get this thing out the door! :p

I think after those PRs we may have all the bugs smashed.

**Comment by jvanz** [comment]
I've took the freedom to create some milestones and add some issues there. Please, take a look and share your thoughts.

**Comment by Eforen** [comment]
> I've took the freedom to create some milestones and add some issues there. Please, take a look and share your thoughts.

Awesome initiative! Thank you! I really need to learn milestones!

**Comment by brannonh** [comment]
Thanks for all your help, guys. Things got really busy for me over the last week or so. I'll be back to helping out as soon as I can.

**Comment by Eforen** [comment]
The main thing is we got those bug fixes out the door so thats good. People can use it.

**Comment by Eforen** [comment]
> Could you please add the option to *not* add the zettel's title as an H1 in the newly created file? It may not be needed for people like me who name zettels with `ID separator title.md`, especially when paired with a plugin like [Embedded note titles](https://github.com/mgmeyers/obsidian-embedded-note-titles).
> 
> Note that this might interfere with the _Insert Zettel Link_ function, which seems to look for H1s.

Please create an issue for this. Also before creating an issue please look at some of the other feature requests for example #13 I believe will solve your use case. If it does please just add a note in that issue about your use case. If not address in your feature request issue why #14 does not solve your use case.

**Comment by Eforen** [comment]
> Thanks for all your help, guys. Things got really busy for me over the last week or so. I'll be back to helping out as soon as I can.

No worries man everyone gets busy.

**Comment by brannonh** [comment]
> No worries man everyone gets busy.
> 
> Sorry if I got a little impatient and pushed the version out the door lolz

No complaints here. 🙂 The whole point of having multiple maintainers is that all of them can keep moving the project forward, right?
